### ISSUE TITLE
Some templates do not work in Dev tools -> Templates, but work in Markdown card

### ISSUE THREAD

**Issue by ildar170975** [PiotrMachowski/Home-Assistant-custom-components-Custom-Templates#14]
Smth strange is happening in 2023.9.2 (may be happened earlier…)

This works:
```
Translated en: {{ "sun.sun" | ct_state_translated("en") }}
Translated ru: {{ "sun.sun" | ct_state_translated("ru") }}
```
![19a931bc90cd47b9e34a8daaba6588fe1d14d769](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/assets/71872483/e5716f16-58b5-48c1-b235-fd6be9c89595)

but this does not:
```
Translated en: {{ ct_state_translated("sun.sun", "en") }}
Translated ru: {{ ct_state_translated("sun.sun", "ru") }}
```
![b9cbd74aa3c5a791cb4fa30aa9152d14c0e73666](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/assets/71872483/428002cb-268d-4cef-aa63-253caaaacfca)

Same for **ct_translated**:

![588f70db5af2fca1a09c7b2bef317c9768ea5dc4](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/assets/71872483/ef6e7350-2285-42e8-8174-ae43ecc56085)
![aaa8bd5a5aacb6bf97591eda62d06ca83aa5898d](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/assets/71872483/1fb21e30-9329-47e4-8fb9-3e21a3098705)

Same for **ct_state_attr_translated**:

![6c113db200fb137ebeb886ba593886829877e726](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/assets/71872483/46f5b032-ded5-45b7-90a4-c60963fb9af8)
![b973309e6c6301fadb67b98b1793994b7db00687](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/assets/71872483/f60ac72a-58fa-4bd0-bcdf-8280480f3206)

Also **ct_all_translations** - does not work:

![0c9643d92c50624d60cdd537ce2bbd17d38bf30d](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/assets/71872483/9089f831-e4b3-41cd-a479-c517fc1e08e8)

For **ct_eval** - none of variants works:

![d61789ce7dcd77f2c5bc1acb1d4b72a371b23048](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/assets/71872483/209696aa-585c-4056-b968-e74fffd5afee)
![05bfba284ee8c7acf9609bbb00642336aef0b25e](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/assets/71872483/601f9dfc-7385-4c7f-9910-fd78dd1b2f59)

------

Now the interesting part.
All this things work in Markdown:
```
**ct_state_translated**
State: {{ states("sun.sun") }}
Translated en: {{ ct_state_translated("sun.sun", "en") }}
Translated en: {{ "sun.sun" | ct_state_translated("en") }}
Translated ru: {{ ct_state_translated("sun.sun", "ru") }}
Translated ru: {{ "sun.sun" | ct_state_translated("ru") }}

**ct_state_attr_translated**
Attribute: {{ state_attr("automation.test", "mode") }}
Translated en: {{ ct_state_attr_translated("automation.test", "mode", "en") }}
Translated en: {{ "automation.test" | ct_state_attr_translated("mode", "en") }}
Translated ru: {{ ct_state_attr_translated("automation.test", "mode", "ru") }}
Translated u: {{ "automation.test" | ct_state_attr_translated("mode", "ru") }}

**ct_translated**
Translated en: {{ ct_translated("component.sun.entity_component._.state.below_horizon", "en") }}
Translated en: {{ "component.sun.entity_component._.state.below_horizon" | ct_translated("en") }}
Translated ru: {{ ct_translated("component.sun.entity_component._.state.below_horizon", "ru") }}
Translated ru: {{ "component.sun.entity_component._.state.below_horizon" | ct_translated("ru") }}

**ct_eval**
{% set template_text = "{{ states('sun.sun') }}" -%}
{{ ct_eval(template_text) }}
{{ template_text | ct_eval }}
```
![194a6d0b4d5d388bc80ac7c76879e6d4b457b111](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/assets/71872483/85ddeb79-e07c-4f9b-b1c8-a20216e12d33)

I did not put a code with ct_all_translations - output is too large.

So, the plugin partly does not work in Dev tools → Template, but seems to work in Markdown.

-------------

config:
```
custom_templates:
  preload_translations:
    - en
    - ru
```

**Comment by PiotrMachowski** [comment]
It seems to be related to cache bypassing added [here](https://github.com/home-assistant/core/blob/2023.9.0/homeassistant/helpers/template.py#L484)

**Comment by PiotrMachowski** [comment]
Fixed in v1.2.2

**Comment by ildar170975** [comment]
Confirmed! thank you!
